ods_ci/tests/Tests/600__ai_apps/680__starburst/682__starburst_enterprise_sm.robot

```diff
@@ -0,0 +1,140 @@
+*** Settings ***
+Library             SeleniumLibrary
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/Page/Operators/ISVs.resource
+Resource            ../../../Resources/Page/ODH/AiApps/ManagedStarburst.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Suite Setup          Starburst Enterprise Suite Setup
+Suite Teardown       Starburst Enterprise Suite Teardown
+
+
+*** Variables ***
+${OPERATOR_NAME}=     starburst-enterprise-helm-operator-rhmp
+${SUBSCRIPTION_NAME}=    starburst-enterprise-helm-operator-rhmp-odsci
+${NAMESPACE}=    ods-ci-starburst
+${CHANNEL}=    alpha
+${CATALOG_SOURCE_NAME}=    redhat-marketplace
+${CATALOG_SOURCE_NAMESPACE}=    openshift-marketplace
+${SEP_ROUTE_NAME}=    web-ui
+${SEP_DISPLAYED_NAME}=    Starburst Enterprise
+${DS_PROJECT_NAME}=    ds-sep
+${DS_WORKBENCH_NAME}=    ds-sep-bench
+${DS_PRJ_DESCRIPTION}=    DS project to test Starburst Enterprise integration
+${NB_IMAGE}=    Minimal Python
```

Comment:
IS there a particular reason you are using minimal notebook image ?

Reply:
Understood. I asked this because, in a standard notebook, Pandas is typically pre-installed in the notebook image. Therefore, in the code below, you won't need to install Pandas unless your code relies specifically on a newer version of the Pandas library. However, in other cases, you might need to install it.